addons/mission/functions/fnc_respiratorEffects.sqf

```diff
@@ -45,12 +48,13 @@ GVAR(oldGlasses) = "";
             playSound "tacr_gasmask_off";
             "tacr_gasmask_overlay" cutFadeOut 0;
         };
+
         // Damage
-        if (ACE_player inArea "Contamination" && {GVAR(maskCounter) + 10 < CBA_missionTime}) then {
+        if (_markers findIf {_player inArea _x} == 0 && {GVAR(maskCounter) + 10 < CBA_missionTime}) then {
```

Comment:
`== 0`? Shouldn't this be `!= -1` or `> -1` or `>= 0`?

Reply:
`findIf` comes back as -1 if not found, otherwise index which if you're only checking one thing would be 0.